README.md

```diff
@@ -255,13 +236,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 	        print("error = ", error)
 	    }
 
-	    // Enable or disable features based on the authorization status.
-	    // if you didn't register for remote notifications above you can call `registerForRemoteNotifications` here
-	    // Klaviyo SDK will automatically update the authorization status on next app launch
-	}
+        // you can check if the permission was granted or not but it's not necessary especially if you are using provisional push notifications
+        DispatchQueue.main.async {
+            // this will trigger the didRegisterForRemoteNotificationsWithDeviceToken method to be called where you can collect the push token
+            UIApplication.shared.registerForRemoteNotifications()
```

Comment:
If `registerForRemoteNotifications` is called outside this block, should we advise them to call `setPushToken` directly here? Maybe that doesn't make any difference.

Reply:
You can't do that since you don't have the token here. The token is only available in the application delegate method - `didRegisterForRemoteNotificationsWithDeviceToken`

Reply:
Oh right 🤦‍♂️